sdcm/cluster.py

```diff
@@ -879,7 +879,7 @@ def _wait_private_ip(self):
     def _refresh_instance_state(self):
         raise NotImplementedError()
 
-    @property
+    @cached_property
```

Comment:
see `refresh_ip_address`, there we need to clear this cache.

I think this how we can clear a cached property:
```
self.__dict__.pop('cql_ip_address', None)
```